CIP-ledger-evolution/STS.md

```diff
@@ -0,0 +1,195 @@
+# The Small Step Semantics Framework for Cardano
```

Comment:
FWIW I'm not sure you necessarily need to have this here. This CIP doesn't need to cover all the information you might need to make a CIP, it's reasonable to expect people to need to refer to other documents. e.g. you're not going to include the full current ledger spec!

Indeed, I'm not sure what this document is _doing_. The section in the main doc that links here is just kind of there - what's it for? Is it background information maybe? Then it should be in the background section.